### ISSUE TITLE
 "Local_file" error during initialization

### ISSUE THREAD

**Issue by sacredx72** [deblockt/hass-proscenic-790T-vacuum#16]
## Bug Report

good afternoon, I saw that your repository was updated, and decided to update my system and your integration, but after rebooting the system I recorded errors in the system log

#### Versions

- Driver: Core
core-2021.8.8
supervisor-2021.06.8

#### Steps to reproduce

```
<details>
<summary>2021-08-24 23:19:32 ERROR (MainThread) [homeassistant.setup] Unable to set up dependencies of proscenic. Setup failed for dependencies: local_file
2021-08-24 23:19:32 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_protocol.py", line 422, in _handle_request
    resp = await self._request_handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 499, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_middlewares.py", line 119, in impl
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 60, in security_filter_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 89, in forwarded_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 24, in request_context_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 78, in ban_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 144, in auth_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 135, in handle
    result = await result
  File "/usr/src/homeassistant/homeassistant/components/config/config_entries.py", line 128, in post
    return await super().post(request)
  File "/usr/src/homeassistant/homeassistant/components/http/data_validator.py", line 63, in wrapper
    result = await method(view, request, *args, **kwargs)
  File "/usr/src/homeassistant/homeassistant/helpers/data_entry_flow.py", line 72, in post
    result = await self._flow_mgr.async_init(
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 153, in async_init
    flow, result = await task
  File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 171, in _async_init
    flow = await self.async_create_flow(handler, context=context, data=data)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 693, in async_create_flow
    await async_process_deps_reqs(self.hass, self._hass_config, integration)
  File "/usr/src/homeassistant/homeassistant/setup.py", line 371, in async_process_deps_reqs
    raise HomeAssistantError("Could not set up all dependencies.")
homeassistant.exceptions.HomeAssistantError: Could not set up all dependencies.
</summary>
```

**Comment by deblockt** [comment]
Hi, 

Can you check if the file `/tmp/proscenic_vacuum_map.svg` exists on your filesystem.
Have you tried to remove the integration and reinstall it using the integration menu?

**Comment by deblockt** [comment]
closed on version 0.0.8
